### PR TITLE
fixed-page-and-leader-pages

### LINKED ISSUE #1919: Fix Communities of Practice page mobile view
### Overview
We need to fix the [Communities of Practice page](https://www.hackforla.org/communities-of-practice)'s mobile view because somehow it has changed and some sections no longer match the original Figma design

### Action Items
- [ ] Ensure that all of the page cards (in mobile) have a consistent image height- they should match the ones in the [Program Areas page](https://www.hackforla.org/program-areas).
   - [ ] The images for the Ops and UI/UX cards are currently too tall (see screenshot below) - they need to match the rest 
- [ ] The leader cards' text in mobile currently looks different from the [original Figma design](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3194%3A25627) (in pink rectangle)- the lines wrap to a second row even though they shouldn't (see screenshot below). Please fix them so that they match the original design

### Resources/Instructions
[Communities of Practice page](https://www.hackforla.org/communities-of-practice)
[Original Figma design](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3194%3A25627) (in pink rectangle)
[Program Areas page](https://www.hackforla.org/program-areas)

<details>
<summary>Page cards images in mobile that are too tall</summary>
<br>
Screen Shot 2021-07-11 at 8 44 02 AM
</details>

<details>
<summary>Current leader cards that don't match the original design (incorrect)</summary>
<br>
Screen Shot 2021-07-11 at 8 47 29 AM
</details>

<details>
<summary>Original Figma design leader cards (correct):</summary>
<br>
Screen Shot 2021-07-11 at 8 47 38 AM
</details>

### PR DESCRIPTION
**Pull request by JasonUranta**
Fixes #1919 

### What changes did you make and why did you make them ?

  - I deleted line: "width: 70%;" in "_sass/components/_projects-page.scss/leader-description/{} @media #{$bp-below-mobile}" because it caused the wrap around of the leader page names.
  - With Bonnie, I changed height of page cards of UI/UX and Ops to make sure they match the height of the rest on "_sass/components/_projects-page.scss/.leader-img--communities"
  -

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
Before:

Screen Shot 2021-07-23 at 9 21 17 PM
Screen Shot 2021-07-23 at 9 21 05 PM
Screen Shot 2021-07-23 at 9 20 59 PM
Screen Shot 2021-07-23 at 9 20 50 PM
Screen Shot 2021-07-23 at 9 20 42 PM
Screen Shot 2021-07-23 at 9 21 10 PM

After:

Screen Shot 2021-07-23 at 9 15 24 PM
Screen Shot 2021-07-23 at 9 15 16 PM
Screen Shot 2021-07-23 at 9 15 06 PM
Screen Shot 2021-07-23 at 9 14 47 PM
Screen Shot 2021-07-23 at 9 17 59 PM

### CODE REVIEW

**Comment by ExperimentsInHonesty** [comment]
@JasonUranta at the top of this pr you need to indicate which issue this fixes (it's the first line).

**Review by adam-abundis — APPROVED** [review]
@JasonUranta I was able to pull your changes and see the changes for the `Communities of Practice` cards in mobile view. It looks really good.

Also, great job providing a summary and before/after screenshots in the pull request. It was really well done. Keep up the good work.

File: _sass/components/_project-page.scss [hunk 1]
```diff
@@ -331,7 +331,7 @@
     min-width: 70%;
     display: flex;
     flex-direction: column;
-    width: 70%;
+
```
Comment (adam-abundis) [inline _sass/components/_project-page.scss]: remove this extra space entirely on line 334

**Review by adam-abundis — CHANGES_REQUESTED** [review]
@JasonUranta For upkeep purposes, let's remove the whitespace in line 334 entirely. This is so the CSS can be more readable.

**Review by JasonUranta — COMMENTED** [review]
Hello! At the request of Mr. Adam Abundis, I deleted the extra line on line 334.

**Review by adam-abundis — APPROVED** [review]
Thank you for the changes @JasonUranta.

**Pull request merged**